projects/distributed-tracing/src/shared/dashboard/widgets/table/table-widget-renderer.component.scss

```diff
@@ -7,6 +7,7 @@
 .table {
   flex: 1 1;
   width: 100%;
```

Comment:
this scares me - shouldn't the table be sized not to overflow?

Reply:
Yeah, its odd. But this is the overflow for the table-widget-renderer. There is another overflow:auto in table itself that usually does the scrolling, but in explorer due to the way the page flexes, it is required here too.

Reply:
Changed it to hidden, because that's really more accurate since it isn't actually doing the scrolling.

Reply:
I pulled it locally and chromium looks fine, but safari doesn't - 
![image](https://user-images.githubusercontent.com/45047841/103953355-d45d6700-50f6-11eb-847f-5a00a13e5423.png)

Reply:
Also, using `min-height: 0;` instead of  the overflow seems to have the same effect in chromium (but also, unfortunately the same effect in safari - perhaps that's a pre-existing issue?)

Reply:
How wonderful. 

Thanks. Checking now.

Reply:
After looking a bit, I'm guessing this must have been existing for a while. We just don't get Safari traffic so it hasn't been reported.

Can we get this fix in to at least get things fixed on Chrome and FF? I'll follow up with a fix for Safari.

Reply:
Yep - do you think it makes min-height instead of overflow here? I was feeling it was better because overflow (auto or hidden) is describing behavior we don't actually want - it should never overflow by design. Min-height works because the issue that I was seeing is the default of min-height:auto in flex col layout was preventing shrinking.

Reply:
Sure, agreed. Let me test that change in the Safari fix and use min-height instead if it works.

Reply:
Think I found a solution that works for Safari. Just going to tack it on to this one.

Reply:
nice!